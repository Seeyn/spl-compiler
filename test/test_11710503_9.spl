struct foo{
    int c,b;
};

int main(){
    int a[2][3];
    struct foo d;
    a[0][0] = 1;
    d.c = a[0][0];
    d.b = 2.0;
}