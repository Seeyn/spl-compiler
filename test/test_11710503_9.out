Error type 5 at Line 10: unmatching type on both sides of assignment
